CHANGELOG.md

```diff
@@ -5,7 +5,8 @@
 
 ### Fixed
 * <How to hit and notice issue? what was the impact?> ([#????](https://github.com/realm/realm-core/issues/????), since v?.?.?)
-* None.
+* Making a query that compares two integer properties could cause a segmentation fault on the server.
```

Comment:
I think this should refer to x86 instead of "on the server"

Reply:
@bmunkholm What would be the preferred term here?